Winapp3/Beta/Winapp3.ini

```diff
@@ -1,26 +1,26 @@
-; Version: 180410
+; Version: 180412
 ; # of entries: 118
 ;
-; Winapp3 is fully licensed under the CC-BY-SA-4.0 license agreement. 
+; Winapp3 is fully licensed under the CC-BY-SA-4.0 license agreement.
 ; If you plan on modifying and/or distrubuting the file in your own program, please ask first.
 ;
-; This is the advanced version of Winapp2, a.k.a Winapp3. This file contains dangerous and aggressive entries. 
+; This is the advanced version of Winapp2, a.k.a Winapp3. This file contains dangerous and aggressive entries.
 ; DO NOT use this, unless you know what you are doing and you know how to back up and restore your system.
 ; This file is completely unpredictable and may even nuke your system. Use with extreme caution!
 ; You have been warned. We are not responsible for damage of anyones system.
 ;
-; The first step would be to rename Winapp3 to Winapp2 so it is useable with all cleaners. 
+; The first step would be to rename Winapp3 to Winapp2 so it is useable with all cleaners.
 ; If you don't know how to do this, then you probably shouldn't be using this file.
 ;
 ; Language entries
 
 [7-Zip Languages *]
-LangSecRef=Language Files
-DetectFile=%ProgramFiles%/7-zip
+Section=Language Files
+DetectFile=%ProgramFiles%\7-zip
 Default=False
 Warning=This will delete all languages excluding English.
-FileKey1=%ProgramFiles%/7-Zip/Lang|*.*
-ExcludeKey1=FILE|%ProgramFiles%/7-Zip/Lang|en.ttt
+FileKey1=%ProgramFiles%\7-Zip\Lang|*.*

```

Comment:
Nice find. Perhaps this is a good suggestion for Winapp2ool to find?

Reply:
Yes, Winapp2ool could check Detects and Keys for the "wrong" slash (/).